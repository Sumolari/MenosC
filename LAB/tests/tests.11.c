{
	int valor;
	bool continuar;
	read( valor );
	continuar = true;
	condicion = continuar == true;
	read( valor );
	for ( continuar = true; continuar == true; continuar = true ) { }

		if ( valor < 3 )
			continuar = false;
		else
			continuar = true;
}