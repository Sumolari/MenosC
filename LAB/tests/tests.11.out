Linea 6: Variable no declarada

Numero de errores:      1
